ONSdigital/dp-areas-api
# update dependencies to use log.go v2

### What

Update ONS libraries to versions using log.go v2:
- dp-healthcheck 1.0.5 -> 1.1.0
- dp-mongodb 2.3.0 -> 2.4.0
- dp-net 1.0.12 -> 1.1.0

Also update log.go 2.0.4 -> 2.0.5

These changes needed to allow logs to be searchable
in Elasticsearch / Kibana (error logs particularly)

### How to review

- check it builds with make build
- run tests with make test
- audit dependencies with make audit
- eyeball changes for any obvious errors

### Who can review

Anyone
